src/common/components/repositories-home/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchUserRepositories } from '../../actions/repositories';
+
```

Comment:
Picky: Can the first-party dependency imports be grouped together without these line breaks?